Lib/test/test_faulthandler.py

```diff
@@ -817,6 +817,17 @@ def test_disable_windows_exc_handler(self):
         self.assertEqual(output, [])
         self.assertEqual(exitcode, 0xC0000005)
 
+    def test_cancel_later_without_dump_traceback_later(self):
+        # bpo-37933: Calling cancel_dump_traceback_later()
+        # without dump_traceback_later() must not gfaust
```

Comment:
There is a typo, no?

```suggestion
        # without dump_traceback_later() must not segfaust
```

Reply:
Note: I'm not allowed to merge my suggestion (push into your branch, from GitHub web UI).